doc/ansible/dev/developer_guide.md

```diff
@@ -390,6 +390,22 @@ can be used as shown:
       foo: bar
 ```
 
+### Ansible Operator Conditions
+The Ansible Operator has a set of conditions which it will use as it performs
+its reconciliation procedure. There are only a few main conditions:
+
+* Running - the Ansible Operator has called Run on the Ansible Runner to invoke
+  your Ansible logic.
+
+* Successful - if the run has finished and there were no errors, the Ansible
+  Operator will be marked as Successful. It will then wait for the next
+  reconciliation period.
```

Comment:
s/next reconciliation period/next reconciliation action, either the reconcile period, dependent watches triggers or the resource is updated.